components/datadog/agent/kubernetes_object_ref.go

```diff
@@ -0,0 +1,55 @@
+package agent
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review. Do look for something to sincerely praise. -->
**👏 praise**
This is awesome, and probably replies to my previous question